lib/Text_Wiki/Text/Wiki/Parse/Default/Separator.php

```diff
@@ -46,7 +46,11 @@ class Text_Wiki_Parse_Separator extends Text_Wiki_Parse {
     *
     */
 
-    public $regex = '/^([=]{4,})$/m';
+    public $regex = '/
+        ^
+        ([=]{4,})	# "====", four or more =
```

Comment:
If this is our parser for <hr> we need to change `[=]` to `[=-]` next.

Reply:
The parser for <hr> is in Horiz.php. This is for content section separators.